src/v2/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -49,6 +53,11 @@ export class ArtworkApp extends React.Component<Props> {
    * data that remains consistent with the rest of the app.
    */
   componentDidMount() {
+    if (this.shouldRenderSubmittedOrderModal()) {
+      // TODO: Look into using router push
+      // this.props.router.replace(this.props.match.location.pathname)
+      window.history.pushState({}, null, this.props.match.location.pathname)
```

Comment:
I wanted to use the router for removing the query param but found no way to keep the page from rerendering. Ended up copying the solution we're using all over Force.